gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -23,6 +24,7 @@
               <span translate ng-if="folderEditor.folderID.$valid || folderEditor.folderID.$pristine">Required identifier for the folder. Must be the same on all cluster devices.</span>
               <span translate ng-if="folderEditor.folderID.$error.uniqueFolder">The folder ID must be unique.</span>
               <span translate ng-if="folderEditor.folderID.$error.required && folderEditor.folderID.$dirty">The folder ID cannot be blank.</span>
+              <span translate ng-show="!editingExisting">When adding a new folder, keep in mind that the Folder ID is used to tie folders together between devices. They are case sensitive and must match exactly between all devices.</span>
```

Comment:
Thanks for reviewing!
I moved it up below the folder ID input because - since sharing is in separate tab - this note is now below folder path input where it does not make sense.
Should I delete it? I'm happy with just "Required identifier for the folder. Must be the same on all cluster devices." in case of folder editing and adding new folder.

Reply:
It didn't make more sense below the "Share With Devices" section, did it? I like the message, I believe it's more user friendly than the "Required ..." one - so I'd like to keep it, but I don't care much whether here or at the bottom as before.